src/trustedtypes.js

```diff
@@ -289,16 +349,35 @@ export const TrustedTypes = (function() {
    * @todo Figure out if the return value (and the builder) can be typed.
    */
   function createPolicy(name, builder) {
+    // TODO(msamuel): could we also allow symbol names for non-exposed policies?
```

Comment:
Not really, for now we're sticking with strings, as there's a policy name whitelist in CSP, and that one relies on strings.

Reply:
Removed